scripts/ci/check_compliance.py

```diff
@@ -606,7 +636,7 @@ def run(self):
         if not self.ensure_npx():
             self.skip(
                 'dts-linter not installed. To run this check, '
-                'install Node.js and then run [npm ci] command inside ZEPHYR_BASE'
+                'install Node.js and then run [npm --prefix ./scripts/ci ci] command inside ZEPHYR_BASE'
```

Comment:
At least the hint command shall be fixed. `npm ci` failed for me as no `package.json` under `ZEPHYR_BASE`.